modules/kubernetes-addons/crossplane/main.tf

```
@@ -18,18 +18,20 @@ module "helm_addon" {
 # AWS Provider
 #--------------------------------------
 resource "kubectl_manifest" "aws_controller_config" {
-  count = var.aws_provider.enable == true ? 1 : 0
+  count = local.aws_provider.enable == true ? 1 : 0
```

I missed where this local is defined - regardless, we should catch attribute lookups in a `try()` in case a value is not provided and set a default
```suggestion
  count = try(local.aws_provider.enable, true) == true ? 1 : 0
```